src/components/PostCard/PostCard.js

```diff
@@ -95,23 +97,27 @@ export default class PostCard extends React.Component {
           </div>
         )}
         {!isEvent && (
-          <PostBody
-            {...post}
+          <div onClick={this.onClick}>
```

Comment:
i know aaron just changed all this stuff, i believe you this could be better but just checking